packages/govern-server/src/api-graphql/schema.ts

```diff
@@ -22,6 +22,7 @@ export default gql`
     registryEntries: [RegistryEntry!]!
     containers: [Container]!
     roles: [Role!]!
+    queues: [GovernQueue]!
```

Comment:
A DAO only has one queue

Reply:
Do you mean that we should enforce this idea, only ever returning the first queue? Technically a DAO can have any number of queues.

Reply:
... sorry forgot that it is possible to have many different queues for an executor...